automation/test.sh

```diff
@@ -173,13 +173,19 @@ export IMAGE_PREFIX_ALT=${IMAGE_PREFIX_ALT:-kv-}
 # probably because load on the node, possible situation when the bazel
 # fails to download artifacts, to avoid job fails because of it,
 # we repeat the build images action
+tries=3
```

Comment:
Can you extract this to a function?
The return value should hopefully be either 0 for success or the last attempt result.